registry/infrastructure/repositories/organization_repository.py

```diff
@@ -56,13 +56,12 @@ def get_groups_for_org(self, org_uuid):
         self.session.commit()
         return groups_domain_entity
 
-    def store_ipfs_hash_and_test_transaction_hash(self, organization, username, test_transaction_hash):
+    def store_ipfs_hash(self, organization, username):
```

Comment:
what was the test_transaction_hash used for before ?

Reply:
it was transaction hash of test network as earlier it was part of the publish to ipfs function. now we store it in consumer.

Reply:
so what was the transaction hash of test network used for ?